### PR TITLE
update integration section

### PR DESCRIPTION
**Pull request by kurau**
Split integration section into separate blocks by language.
Remove glossary.
Update overview section.

### CODE REVIEW

File: docs/2.0/index.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ include::getstarted.adoc[leveloffset=+1]
 
 include::overview.adoc[leveloffset=+1]
 
-include::glossary.adoc[leveloffset=+1]
+//include::glossary.adoc[leveloffset=+1]
```
Comment (baev) [inline docs/2.0/index.adoc]: remove this line

**Comment by eroshenkoam** [comment]
Good job!

File: docs/2.0/aboutdoc.adoc [hunk 2]
```diff
@@ -25,4 +25,12 @@ There are several places to getting help:
  * Go to https://docs.qameta.io/allure/latest/#_get_started[Get started] to build report for existing project
  * Learn more about report structure and features in this
  https://docs.qameta.io/allure/latest/#_report_structure[section]
- * https://docs.qameta.io/allure/latest/#_integration[Integrate] Allure with your testing framework
\ No newline at end of file
+ * Choose your favorite testing framework and integrate it with Allure.
+ Supported frameworks are divided into language-based sections: https://docs.qameta.io/allure/latest/#_java[Java],
+  https://docs.qameta.io/allure/latest/#_python[Python],
```
Comment (lanwen) [inline docs/2.0/aboutdoc.adoc]: just `<<python,Python>>` and so on

Comment (kurau) [inline docs/2.0/aboutdoc.adoc]: thx!

**Pull request merged**